### DISCUSSION TITLE
Filter buildings in cityJson?

### DISCUSSION THREAD

**Discussion by pcace** [cityjson/specs#159] [Q&A] [answered]
Hi,

what is the easiest way to filer a cityJson by building (for example with a certain position) using a shell/typscript/python?

Any help would be awesome! Thanks a lot!

**Comment by hugoledoux** [comment, marked as answer]
1. install [cjio](https://github.com/cityjson/cjio/)
2. find the ID of the building you want to filter, click on it in qgis should give it to you
3. `cjio myfile.json subset --id theID save mybuilding.json`

**Reply by pcace** [reply]
thank you so much for your fast reply! if i would like to filter buildings by coordinate (without qgis, because it all needs to happen in a script without UI) any suggestion where i would start then? 

Cheers and thanks!

**Reply by hugoledoux** [reply]
cjio subset has a bbox option

**Reply by pcace** [reply]
> cjio subset has a bbox option

Awesome! i must have missed that in the documentation! Thanks a lot - exactly what i was looking for!
